src/System.Private.CoreLib/shared/System/Globalization/CultureNotFoundException.cs

```diff
@@ -89,7 +90,7 @@ private static string DefaultMessage
             }
         }
 
-        private string FormatedInvalidCultureId
+        private string? FormatedInvalidCultureId
```

Comment:
OPtional: fix spelling FormatedInvalidCultureId 😄

Reply:
I'll leave it to not cause potential serialization issues

Reply:
> I'll leave it to not cause potential serialization issues

Exception implements ISerializable, which means BinaryFormatter won't look at its fields.  It should be fine to change it.
cc: @ViktorHofer

Reply:
Oh, and this is a property... that wouldn't ever be considered by BinaryFormatter.

Reply:
Correct, exceptions implement ISerializable and only respect GetObjectData.